frontend/tests/components/draft/sidebar.test.tsx

```
@@ -92,113 +103,144 @@ describe("Sidebar", () => {
     });
   });
 
   it("renders droplet name and lessons", () => {
-    render(
-      <Sidebar
-        user={mockUser as any}
-        droplet={mockDroplet as any}
-        authorizedUser={mockAuthorizedUser as any}
-      />,
-    );
+    render(<Sidebar user={mockUser as any} droplet={mockDroplet as any} />);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)